playbooks/cluster_setup_debian_centos.yaml

```diff
@@ -0,0 +1,71 @@
+# This playbook setup SEAPATH on both debian and CentOS machines
+# The machine must previously have been flashed
+# with an iso created by build_debian_iso or CentOS with appropriate kickstart file
+# This playbooks can also configure VMs created with build_debian_iso
+
+- name: Check if we are using a correct seapath flavor
+  hosts:
+    - cluster_machines
+    - standalone_machine
+    - VMs
+  tasks:
+    - name: Retrieve redhat-release
```

Comment:
Ansible facts and `ansible_distribution`

Reply:
Fixed